libr/io/section.c

```diff
@@ -277,8 +277,6 @@ R_API ut64 r_io_section_vaddr_to_offset(RIO *io, ut64 vaddr) {
 	RIOSection *s;
 
 	r_list_foreach (io->sections, iter, s) {
-		if (!s->vaddr)
```

Comment:
Why a section with vaddr 0 shouldn't be considered? It is possible to have some sections @ 0 (see avr test case).